zuul.d/pipelines.yaml

```diff
@@ -21,7 +21,7 @@
       github:
         status: pending
         comment: false
-        status-url: https://status.openlabtesting.org/status
+        status-url: https://status.openlabtesting.org/status/change/{change.number},{change.patchset}
```

Comment:
@Yikun have committed the changes in PR https://github.com/theopenlab/labkeeper/pull/163/files and have applied it in OpenLab CI zuul node.

@Yikun please help to review and make sure whether we need the updates, thank you

Reply:
@Yikun got it, it's `pending` job status url, thank you @wangxiyuan and @Yikun , LGTM